crates/sui-source-validation-service/config.toml

```diff
@@ -2,7 +2,21 @@
 source = "Repository"
 [packages.values]
 repository = "https://github.com/mystenlabs/sui"
-branch = "mainnet-v1.3.1"
+branch = "framework/mainnet"
+network = "Mainnet"
```

Comment:
Is the network case sensitive? (I would vote for not)

Reply:
fixed up